sources/Renderer/Direct3D12/D3D12RenderSystem.cpp

```diff
@@ -527,6 +533,20 @@ void D3D12RenderSystem::CreateDevice()
     }
 }
 
+void D3D12RenderSystem::CreateDxcInterface()
+{
+    dxcModule_ = LoadLibrary(L"dxcompiler.dll");
```

Comment:
This one should use the `TEXT()` macro instead of explicit wide character string literals in case someone disables the `UNICODE` macro on Windows.

Reply:
Gotchya! I was wondering about that - I'll tweak that here in a moment.